ballerina/build.gradle

```diff
@@ -238,7 +238,14 @@ task stopTestDockerContainers() {
     }
 }
 
-updateTomlFiles.dependsOn copyStdlibs
+def checkForBreakingChanges = (System.getenv('CHECK_BRAEKING_CHANGES') == "true")
+
+if (checkForBreakingChanges) {
+    updateTomlFiles.dependsOn copyToLib
+} else {
+    updateTomlFiles.dependsOn copyStdlibs
+}
```

Comment:
Cant this be moved to the plugin? i.e if the env variable is true we skip "copyStdLibs"

Reply:
Yes, the copyToLib task won't be registered as gradle task when env variable set to true. This if condition is to avoid calling an unregistered task.  Logic`updateTomlFiles.dependsOn copyToLib` couldn't be moved directly to gradle puling where updateTomlFiles task defined in the gradle files of the stdlibs.

Reply:
Noted. Suggestion updated